parkhangs-webapp/src/components/ParkFilterToolbar.js

```diff
@@ -36,32 +42,66 @@ class ParkFilterToolbar extends React.Component {
             && this.state.maxSizeValid
     }
 
+    validateDistance = () => {
+        if (this.state.distance === '' && this.state.lat === '' && this.state.lon === '') {
+            return true
+        }
+
+        if (this.state.distance >= 0 && (!this.state.lat || !this.state.lon)) {
+            return false
+        }
+
+        if (this.state.lat && (!this.state.lon || this.state.distance === '')) {
+            return false
+        }
+
+        if (this.state.lat && (!this.state.lon || this.state.distance === '')) {
```

Comment:
These two seem to be the same checks? For lines 54 and 58

Reply:
I think you're right, I was struggling big time here haha

Reply:
Understandable, you did a lot and this is going to be really cool!

Reply:
fixed it!